code/game/machinery/transportpod.dm

```diff
@@ -0,0 +1,109 @@
+/obj/machinery/transportpod
+	name = "Ballistic Transportation Pod"
+	desc = "A fast transit ballistic pod used to get from one place to the next. Batteries not included!"
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "borg_pod_opened"
+
+	density = 1 //thicc
+	anchored = 1
+	use_power = 0
+
+	var/used = 0
+	var/in_transit = 0
+	var/mob/occupant = null
+
+	var/xc = list(137, 209, 163, 110, 95, 60, 129, 201) // List of x values on the map to go to.
+	var/yc = list(134, 99, 169, 120, 96, 122, 189, 219) // List of y values on the map to go to.
+
+/obj/machinery/transportpod/process()
+	if(used == 1) // Just to make sure.
+		return
+
+	if(occupant)
+		if(in_transit)
+			var/locNum = rand(0, 7) //pick a random location
+			var/turf/L = locate(xc[locNum], yc[locNum], 1) // Pairs the X and Y to get an actual location.
+			doExplosion(L)
+			sleep(15) //Give explosion time so the pod itself doesn't go boom
+			src.forceMove(L)
+			in_transit = 0
+			used = 1
+
+
+/obj/machinery/transportpod/examine(mob/user)
+	..(user)
+	user << "The pod is [(used == 1)? "already used." : "ready to be used."]"
+
+/obj/machinery/transportpod/relaymove(mob/user as mob)
+	if(user.stat)
+		return
+	go_out()
+	return
+
+/obj/machinery/transportpod/update_icon()
+	..()
+	if(occupant)
+		icon_state = "borg_pod_closed"
+	else
+		icon_state = "borg_pod_opened"
+
+/obj/machinery/transportpod/Bumped(var/mob/living/O)
+	if(used)
+		return
+	go_in(O)
+
+/obj/machinery/transportpod/proc/go_in(var/mob/living/carbon/human/O)
+	if(used)
+		return
+
+	if(occupant)
+		return
+
+	if(O.incapacitated()) //aint no sleepy people getting in here
+		return
+
+	add_fingerprint(O)
+	O.reset_view(src)
+	O.forceMove(src)
+	occupant = O
+	update_icon()
+	if(alert(O, "Are you sure you're ready to launch?", , "Yes", "No") == "Yes")
+		playsound(src, HYPERSPACE_WARMUP)
+		in_transit = 1
+	else
+		go_out()
+	return 1
+
+/obj/machinery/transportpod/proc/go_out()
+	if(!occupant)
+		return
+
+	occupant.forceMove(src.loc)
+	occupant.reset_view()
+	occupant = null
+	update_icon()
+
+/obj/machinery/transportpod/verb/move_eject()
+	set category = "Object"
+	set name = "Eject Pod"
+	set src in oview(1)
+
+	if(usr.incapacitated())
+		return
+
+	go_out()
+	add_fingerprint(usr)
+	return
+
+/obj/machinery/transportpod/verb/move_inside()
+	set category = "Object"
+	set name = "Enter Pod"
+	set src in oview(1)
+
+	if(usr.incapacitated()) //just to DOUBLE CHECK the damn sleepy people don't touch the pod
+		return
+
+	go_in(usr)
+
+/obj/machinery/transportpod/proc/doExplosion(var/turf/L) //boom
+	explosion(L, 0, 3, 3)
```

Comment:
This is just a placeholder until I'm not too tired to add in an actual cool effect.